states/huntsman/calibrating.py

```diff
@@ -4,116 +4,87 @@
 from pocs.utils import current_time
 
 
-def wait_for_sun_alt(pocs,
-                     min_altitude=None,
-                     max_altitude=None,
-                     delay=None,
-                     message="Waiting for Sun altitude. Current: {}"):
-    """
-    Wait for the altitude of the Sun to be within given limits
-
-    Args:
-        pocs :
-        min_altitude (astropy.units.Quantity, optional):
-        max_altitude (astropy.units.Quantity, optional):
-        message (str):
-    """
-    if min_altitude is None and max_altitude is None:
-        raise ValueError("At least one of min_altitude & max_altitude must be given")
-    if min_altitude is None:
-        min_altitude = -90
-    if max_altitude is None:
-        max_altitude = 90
-    if isinstance(min_altitude, u.Quantity):
-        min_altitude = min_altitude.to(u.degree).value
-    if isinstance(max_altitude, u.Quantity):
-        max_altitude = max_altitude.to(u.degree).value
-
-    if not delay:
-        delay = pocs._safe_delay
-
-    while pocs.is_safe():
-        sun_pos = pocs.observatory.observer.altaz(current_time(),
-                                                  target=get_sun(current_time())
-                                                  ).alt
-        if sun_pos.value > max_altitude or sun_pos.value < min_altitude:
-            # Check simulator for 'night'
-            if 'night' in pocs.config['simulator']:
-                pocs.logger.info(f'Using night simulator, pretending sun down.')
-                break
-            else:
-                pocs.say(message.format(sun_pos.value))
-                pocs.sleep(delay=delay)
+def _wait_for_twilight(pocs, horizon):
+    '''
+    Wait for twighlight if safe to do so.
+    '''
+    delay = pocs._safe_delay
+
+    while pocs.is_safe(horizon='flat'):
+
+        if not pocs.is_dark(horizon=horizon):
+            pocs.say('Not dark enough for coarse focusing. Waiting...')
```

Comment:
```suggestion
            pocs.say(f'Not dark enough for {horizon}. Waiting...')
```

Or something similar. This is also used for waiting on observe.

Reply:
This gets modified in an upcoming PR